docs/services/sdk/serialization.mdx

```diff
@@ -0,0 +1,108 @@
+---
+sidebar_position: 9
+description: "S"
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Serialization
+
+For some SDK operations, Restate needs to send data over the network, for example for storing state, doing side effects and awakeables.
+
+<Tabs groupId="sdk">
+<TabItem value="ts" label="Typescript">
+
+The SDK serializes values with `Buffer.from(JSON.stringify(payload))` and deserializes them with `JSON.parse(result.toString()) as T`.
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+    
+There are multiple ways to specify which (de)serializers should be used.
+
+** Restate's `CoreSerdes` for primitive types **
+
+For serializing primitive types, use Restate's built-in `CoreSerdes`.
+The following serializers are included:
+
+```java
+CoreSerdes.VOID
+CoreSerdes.BYTES
+CoreSerdes.STRING_UTF8
+CoreSerdes.BOOLEAN
+CoreSerdes.BYTE
+CoreSerdes.SHORT
+CoreSerdes.INT
+CoreSerdes.LONG
+CoreSerdes.FLOAT
+CoreSerdes.DOUBLE
+```
+
+For example, to specify the serializer for state of type Long:
+
+```java
+StateKey<Long> STATE_KEY = StateKey.of("my-key", CoreSerdes.LONG);
+```
+
+** Restate's `CoreSerdes` for Protobuf **
+
+You can also use Restate's built-in `CoreSerdes` to create serializers for Protobuf.
+For example, for the Protobuf message type `GreetingResponse`, we can create a serializer as follows:
+
+```java
+CoreSerdes.ofProtobuf(GreetingResponse.parser());
+```
+
+** Jackson serialization **
+
+If you want to use [Jackson Databind](https://github.com/FasterXML/jackson) to (de)serialize POJOs, add the dependency sdk-serde-jackson.
+For example, in Gradle:
+
+```kotlin
+implementation("dev.restate.sdk:sdk-serde-jackson:1.0-SNAPSHOT")
```

Comment:
I think the group id is now just `dev.restate` here and everywhere we mention our SDK artifacts.